src/select.coffee

```diff
@@ -409,9 +415,9 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
       value = content.objectAt index
     else # setter
       index = content.indexOf value
-      @setHighlightedIndex index, yes
+      @setHighlightedIndex index, @get('shouldEnsureVisible')
```

Comment:
no both of them are highlighted, the 'shouldEnsureVisible' is a flag to control whether the option with the `index` is visible in the dropdown. See the doc for the 'shouldEnsureVisible' variable.